fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/responses/ViewHelpers.java

```diff
@@ -399,7 +399,17 @@ public Resource rdfsClass() {
      * @return content-bearing node for the given subject
      */
     public static Node getContentNode(final Node subject) {
-        return subject == null ? null : createURI(subject.getURI().replace("/" + FCR_METADATA, ""));
+        return subject == null ? null : NodeFactory.createURI(subject.getURI().replace("/" + FCR_METADATA, ""));
```

Comment:
It seems like the content of this change is exactly the opposite of what would seem to be a very natural suggestion from @acoburn. I wouldn't be all that surprised if @awoods had some good reason to do this, but the very whimsicality of our arrangements here bespeaks an ominous lack of gravity in their foundations.